mission-vision-values.md

```diff
@@ -116,3 +116,75 @@ to degrade gracefully as needed.
 OpenTelemetry users should not have to choose between high-quality telemetry and
 a performant application. High performance is a requirement for OpenTelemetry,
 and unexpected interference effects in the host application are unacceptable.
+
+## Community values &mdash; the principles that guide our interactions
+
+The OpenTelemetry project aims to be a welcoming place where new and existing
+members feel safe to respectfully share their opinions and disagreements. We
+want to attract a diverse group of people to collaborate with us, which means
+acknowledging that people come from different backgrounds and cultures. 
+
+There might be situations where community members act in a dubious manner. If
+you have seen or experienced unacceptable behavior or anything that would make
+our community less welcoming, please speak up! See [our code of
+conduct](./code-of-conduct.md) for more information on how to report bad behavior.
+
+While we want to encourage everyone to express themselves in their own way,
+there are some behaviors that we encourage you to adopt while interacting with
+other community members.
+
+#### Act on behalf of the project
```

Comment:
mega-nitpick, sorry: in the sections above, we used `###` (three levels) for this next level of headers, but here we use `####` (four levels). we should be consistent...

Reply:
I like mega-nitpick, keep them coming.